endoreg_db/models/__init__.py

```
@@ -1,396 +1,8 @@
-    "RawPdfFile",
-    "RawVideoFile",
-    "VideoSegmentationLabelSet",
-    "RequirementSet",
-    "RequirementSetType",
-    "Requirement",
-    "RequirementOperator",
-    "RequirementType",
-    "Risk",
-    "RiskType",
-]
+from .administration import *
```

Ruff (0.8.2)

`from .administration import *` used; unable to detect undefined names

(F403)